chart/templates/NOTES.txt

```diff
@@ -6,4 +6,9 @@ Check out our docs at https://rancher.com/docs/rancher/v2.x/en/
 
 Browse to https://{{ .Values.hostname }}
 
+If this is your first time login, use boostrap username and password below:
```

Comment:
boostrap typo. Probably better to rephrase to "If this is the first time you installed Rancher, use username and password as shown below:"